draft-ietf-acme-acme.md

```diff
@@ -2876,7 +2894,7 @@ Label column of the ACME Identifier Types registry.
 Template:
 
 * Label: The identifier for this validation method
-* Identifier Type: The type of identifier that this method applies to
+* Identifier Type: The type of identifier that this method applies to.  For non-ACME validation methods, this field should be set to "N/A".
```

Comment:
Should non-ACME validation methods require a non-ACME identifier type? I thought you would be using non-ACME validation methods to validate DNS type identifiers.

Reply:
I guess that's true in principle, but you're not doing it **in ACME**, so it doesn't seem crucial to specify an ACME identifier type.  That's probably parsing things too closely, though.  I'll change back to "RESERVED".